tests/utils.go

```diff
@@ -2061,6 +2059,19 @@ func createNamespaces() {
 	}
 }
 
+func NewMinimalVMIWithNS(namespace, name string) *v1.VirtualMachineInstance {
```

Comment:
I think we could further decompose this function and merge it eventually into `NewRandomVMIWithNS`.

Our usage was already pretty confusing before and we certainly never want to create VMs with 8Gi in our e2e tests.

To keep the slirp test working, the only thing which has to be done, is ensuring that 

```
	vmi.Spec.Domain.Devices = v1.Devices{Interfaces: []v1.Interface{{Name: "default",
		InterfaceBindingMethod: v1.InterfaceBindingMethod{
			Masquerade: &v1.InterfaceMasquerade{}}}}}
	vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
```

gets unset, so that we try to add the default network. @zhuchenwang, @xpivarc  what do you think?

Reply:
I agree with the further clean-up.

Reply:
Done in the latest commit. Thanks.